.github/workflows/trunk-upgrade.yaml

```
@@ -0,0 +1,25 @@
+name: Weekly Trunk Upgrade
+on:
+  schedule:
+    # Every Monday @ 5am
+    - cron: 0 5 * * 1
+    # Allows us to manually run the workflow from Actions UI
+  workflow_dispatch: {}
+permissions: read-all
+jobs:
+  trunk_upgrade:
+    name: Upgrade Trunk
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # For trunk to create PRs
+      pull-requests: write # For trunk to create PRs
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Trunk Upgrade
+        uses: trunk-io/trunk-action/upgrade@v1
+        with:
+          base: main
+          reviewers: "@masterpointio/Masterpoint Internal"
```

Can't tell if this will work since there's a space, but it's also a string. If this doesn't work, then `masterpoint-internal` will

---

@oycyc I believe it'll prefer `masterpoint-internal`. Let's try that first and then try this after if that doesn't do the trick. 
```suggestion
          reviewers: "@masterpointio/masterpoint-internal"
```